src/krux/pages/wallet_settings.py

```diff
@@ -83,6 +83,18 @@ def load_passphrase_menu(self, mnemonic):
             if passphrase in (ESC_KEY, MENU_EXIT):
                 return None
 
+            # Decode passphrase in case it's a "bytes" object
+            if isinstance(passphrase, bytes):
+                try:
+                    passphrase = passphrase.decode()
+                except:
+                    self.flash_error(t("Failed to load"))
+                    continue
+            # Check if passphrase string is within ascii range
+            if any(byte > 127 for byte in passphrase.encode()):
```

Comment:
I think here it is byte > 126 as ord(~) == 126

Reply:
Did a PR atop @jdlcdl 's with the change to 126

Reply:
right, its the unprintable DEL